yamale/tests/test_command_line.py

```diff
@@ -58,13 +50,13 @@ def test_external_glob_schema(parser):
         os.path.join(dir_path, 'command_line_fixtures/schema_dir/ex*.yaml'), 1, parser)
 
 
-def test_empty_schema_file():
-    try:
+def test_empty_schema_file(capsys):
+    with pytest.raises(ValueError, match='Validation failed!'):
         command_line._router(
-            'yamale/tests/command_line_fixtures/empty_schema',
+            'yamale/tests/command_line_fixtures/empty_schema/data.yaml',
```

Comment:
Nice find! This test was missing the final `assert False`. Please add the `data.yaml` file to git. Thanks!